devenv/dev-dashboards/Makefile

```diff
@@ -0,0 +1,9 @@
+DASHBOARDS = $(shell find ./ -type f -name '*.json')
+
+vendor:
+	jb install
```

Comment:
do we need to add jb to bingo?

Reply:
good catch! i'll add it if it's not there

Reply:
added with https://github.com/grafana/grafana/pull/57928/commits/21f12a6c9a14a756b4bcbe68adfea0a519f2d872